google/pubsub/v1/pubsub_gapic.yaml

```diff
@@ -434,7 +434,7 @@ interfaces:
     max_retry_delay_millis: 60000 # 60 seconds
     initial_rpc_timeout_millis: 12000 # 12 seconds
     rpc_timeout_multiplier: 1
-    max_rpc_timeout_millis: 12000 # 12 seconds
+    max_rpc_timeout_millis: 30000 # 30 seconds
```

Comment:
This means that GAPIC will be killing the stream every 30 seconds. Can we get this closer to the 10 minutes total timeout?

Do we have an existing issue to not kill long-living streams?